.github/workflows/rust.yml

```diff
@@ -171,6 +171,43 @@ jobs:
         run: |
           ./ohos-build cargo build --target="${{ matrix.target }}"
 
+  ohos-release:
+    name: "OpenHarmony release artifact"
+    runs-on: ubuntu-latest
+    # if: ${{ github.ref == 'refs/heads/main' && github.event_name == 'push' }}
+    permissions:
+      id-token: write
+      contents: read
+      attestations: write
+    strategy:
+      matrix:
+        target: [ 'aarch64-unknown-linux-ohos', 'x86_64-unknown-linux-ohos' ]
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup OpenHarmony SDK
+        id: setup_sdk
+        uses: openharmony-rs/setup-ohos-sdk@v0.1
+        with:
+          version: "4.1"
+      - name: Install Rust
+        uses: dtolnay/rust-toolchain@stable
+        with:
+          targets: ${{ matrix.target }}
+      - name: Build (arch ${{ matrix.target }} )
+        env:
+          OHOS_SDK_NATIVE: ${{ steps.setup_sdk.outputs.ohos_sdk_native }}
+        run: |
+          ./ohos-build cargo build --release --target="${{ matrix.target }}"
+#      - name: Generate artifact attestation
```

Comment:
Could someone with permissions comment this block out and push the result to a branch in the main repo, so we can test that the attestion works as expected before merging?